notebooks/CLT/CLT-006/CLT-006-ComCam.ipynb

```diff
@@ -0,0 +1,338 @@
+{
```

Comment:
<p>I see an empty cell in TimesSquare at the end. Otherwise looks good! </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>